.gitignore

```diff
@@ -55,3 +55,7 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Misc
```

Comment:
These can be placed outside the betelgeuse repo, since it is not related to the project is better to remove the ignore and use on the home for .pylarion and clone pylarion elsewhere.